### ISSUE TITLE
Pass className to Selector

### ISSUE THREAD

**Issue by justemoilouise** [charkour/react-reactions#114]
Ideally, it would be nice to be able to pass a `className` into the selectors so users could add their own styles into it - background-color, etc.

![Screen Shot 2021-09-09 at 6 32 23 PM](https://user-images.githubusercontent.com/6330131/132670635-c83ee1bb-2ce9-4e4d-b804-422230c15f67.png)

**Comment by charkour** [comment]
I agree. Want to make a PR? I'm a little busy this week so it would be a bit until I can add this.

**Comment by justemoilouise** [comment]
@charkour I'd be happy to do it. I'll create one as soon as I have time. 😁

**Comment by charkour** [comment]
Thanks! I appreciate it.

**Comment by charkour** [comment]
@justemoilouise, which selectors would you like this to apply to? All of them?

Also, are you actively working on this? If yes, I can assign you to this issue. Thanks

**Comment by justemoilouise** [comment]
@charkour it would be nice to apply it to all of them. I haven't started working on this. 😅

**Comment by charkour** [comment]
Alright, thanks for the update!

[ ] Facebook Selector (and Pokemon Selector)
[ ] GitHub Selector
[ ] Slack Selector

We can use this to track updates. I'm not sure when I'll get time to do this either. But I'll update this if I start to work on it. Or you can let me know if you do. Thanks!!

**Comment by lagroms** [comment]
Hello,

Is there any news regarding the custom styling feature ?

Thanks !

**Comment by charkour** [comment]
Hey @lagroms, 

Right now this custom styling feature is supported for the Custom Selector. Do you need support for other selectors?

Thanks

**Comment by lagroms** [comment]
Well I'm using the custom ReactionBarSelector, but no custom class in being applied in my case. Maybe I'm missing something ? 

I'm using css-modules in my case : 

```
const Reactions = ({ onSelect }) => {
    return (
        <ReactionBarSelector
            className={styles.reactions_container}
            reactions={reactionOptions}
            onSelect={onSelect}
        />
    );
};
```

**Comment by lagroms** [comment]
My bad, my question was regarding the ability to add a className on the <ReactionBarSelector /> component, instead of passing the style directly to it, and also have more control on the rest.

For exemple, i'd like to style the labels, not sure this is currently possible ? 

Thanks.

**Comment by charkour** [comment]
Hi @lagroms! 

Currently it is not possible to style the labels, but that is a great idea. Would you be willing to make a PR for this feature?
